packages/core/src/browser/browser-clipboard-service.ts

```diff
@@ -84,13 +84,12 @@ export class BrowserClipboardService implements ClipboardService {
                     It can be enabled by 'dom.events.testing.asyncClipboard' preference on 'about:config' page. Then reload Theia.
                     Note, it will allow FireFox getting full access to the system clipboard.`);
                 }
-                throw new Error('Failed writing the the clipboard.');
+                throw new Error('Failed writing the clipboard.');
             }
         }
         if (permission.state === 'denied') {
             // most likely, the user intentionally denied the access
-            this.messageService.error("Access to the clipboard is denied. Check your browser's permission.");
-            throw new Error('Access to the clipboard is denied.');
+            this.messageService.warn("Access to the clipboard is denied. Check your browser's permission.");
```

Comment:
Is not it confusing that we use the same warning message for read and write? Would not it better to have `Read access` and `Write access` instead of just `Access`?

Reply:
But there're no separate read/write settings available to the user. It's just general access.

Reply:
ok, I was concerned to know from where an error is coming, right now it is not clear whether extension tried to read or write. probably it is not important, let's wait with it.